index.js

```diff
@@ -53,6 +55,9 @@ function Store (dbName, options) {
       isConnected: syncApi.isConnected
     }
   )
+  // passes the existing api methods and a new version of pouchdb-hoodie-api with a new EventEmitter internally
+  api.scoped = scoped.bind(this, api)
```

Comment:
the comment says we'd pass in a new version of pouchdb-hoodie-api and a new EventEmitter, but I don't see that happening in the line below?